src/main/java/com/palantir/gerrit/gerritci/ui/client/ProjectSettingsScreen.java

```diff
@@ -327,6 +430,10 @@ public void onClick(ClickEvent event) {
                         params.put("publishBranchRegex", makeXMLFriendly(publishBranchRegex.getText()));
                         params.put("publishCommand", makeXMLFriendly(publishCommand.getText()));
 
+                        params.put("cronJobEnabled", cronJobEnabled.getValue());
+                        params.put("cronCommand", makeXMLFriendly(cronCommand.getText()));
+                        params.put("cronJob", makeXMLFriendly(cronJob.getText()));
+
```

Comment:
Jenkins treats a cron job similar to any other job. So the user can specify or Jenkins will choose based on the change